RoadMap_SwiftUI/RoadMap_SwiftUI/View/LandingView.swift

```diff
@@ -77,10 +79,10 @@ struct LandingView: View {
             }
         }
     }
-
+    
     private var getStartedButtonView: some View {
         NavigationLink(isActive: $viewModel.isDescriptionViewShown) {
```

Comment:
```suggestion
        NavigationLink(isActive: $viewModel.isMainTabViewShown) {
```
так как поменял экран на который переходишь по нажатию кнопки, то замени и название флага